distribution/lib/Standard/Test/0.0.0-dev/src/Extensions.enso

```diff
@@ -294,15 +297,20 @@ Error.should_equal self that frames_to_skip=0 =
             1.00000001 . should_equal 1.00000002 epsilon=0.0001
 Number.should_equal : Float -> Float -> Integer -> Spec_Result
 Number.should_equal self that epsilon=0 frames_to_skip=0 =
-    matches = case that of
```

Comment:
This change is way easier of parse, we the _function signature_ clearly stays unaffected.